src/site/content/en/blog/flexbox-gap/index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Chromium Lands Flexbox `gap`
+subhead: Chromium's CSS Flexbox and Multi-Column layout engines now support gap. `grid-gap` is old news. 
+authors:
+  - adamargyle
+  # - dgrogan
+description: Chromium's CSS Flexbox and Multi-Column layout engines now support gap. `grid-gap` is old news. 
+date: 2020-04-30
+updated: 2020-04-30
+tags:
+  - layout
+  - flexbox
+  - css
+  - gap
+# hero: hero.jpg
+alt: TODO
+---
+
+CSS Grid brought a delightful spacing feature called [`grid-gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap), which quickly became a popular way to put space into layouts. It was easy to fall in love with because it took upon itself so much responsibility with such elegance. Starting in Chromium 85 `grid-gap` is now just [`gap`](https://drafts.csswg.org/css-align-3/#propdef-gap), and you can use it with Flexbox. 💪
+
+## Browser compatibility
+
+At the time of writing `gap` is supported in desktop Firefox (version 63), Firefox for Android (version 63), and Chromium-based browsers (version 85). See [Browser compatibility](https://developer.mozilla.org/docs/Web/CSS/gap#Browser_compatibility) for updates.
+
+```css/3-4/1-2
+.grid {
+  display: grid;
+  grid-gap: 10px;
+  display: flex;
+  gap: 10px;
+}
+```
+
+## CSS Gap
+
+{% Aside 'key-term' %}
+`gap` is the spacing *between children*. You may have heard of this type of spacing being called "gutters" or "alleys". It's space only where the children box edges touch.
+{% endAside %}
+
+`gap` is [flow relative](https://www.w3.org/TR/css-logical-1/#intro), meaning it changes dynamically based on the direction of content flow. For example, `gap` will automatically adjust for the different`writing-mode` or `direction` values that you set for your international users. This significantly eases the burden of spacing challenges for the component and CSS author. **Less code scaling further.**
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <!-- <source src="https://storage.googleapis.com/web-dev-assets/portals_vp9.webm" type="video/webm; codecs=vp8"> -->
```

Comment:
Do you need me to upload this? Link me to the webm file on Drive if so, plz

Reply:
i can upload, i need to convert

Reply:
unless you wanna download the mp4, convert, and upload for me? i'm equipped for the task, but happy to get help?

Reply:
I tried to halp but `ffmpeg` is taking forever on my machine